modules/eks/eks-without-spotinst/variables.tf

```diff
@@ -0,0 +1,346 @@
+    resources_to_tag           = null
+    tags                       = null
+  }
+}
+
+variable "iam_roles_environment_name" {
+  type        = string
+  description = "The name of the environment where the IAM roles are provisioned"
+  default     = "gbl"
+}
+
+variable "iam_primary_roles_stage_name" {
+  type        = string
+  description = "The name of the stage where the IAM primary roles are provisioned"
+  default     = "identity"
+}
+
+variable "iam_primary_roles_tenant_name" {
+  type        = string
+  description = "The name of the tenant where the IAM primary roles are provisioned"
+  default     = null
+}
+
+variable "cluster_encryption_config_enabled" {
+  type        = bool
+  default     = true
+  description = "Set to `true` to enable Cluster Encryption Configuration"
+}
+
+variable "cluster_encryption_config_kms_key_id" {
+  type        = string
+  default     = ""
+  description = "KMS Key ID to use for cluster encryption config"
+}
+
+variable "cluster_encryption_config_kms_key_enable_key_rotation" {
+  type        = bool
+  default     = true
+  description = "Cluster Encryption Config KMS Key Resource argument - enable kms key rotation"
+}
+
+variable "cluster_encryption_config_kms_key_deletion_window_in_days" {
+  type        = number
+  default     = 10
+  description = "Cluster Encryption Config KMS Key Resource argument - key deletion windows in days post destruction"
+}
+
+variable "cluster_encryption_config_kms_key_policy" {
+  type        = string
+  default     = null
+  description = "Cluster Encryption Config KMS Key Resource argument - key policy"
+}
+
+variable "cluster_encryption_config_resources" {
+  type        = list(any)
+  default     = ["secrets"]
+  description = "Cluster Encryption Config Resources to encrypt, e.g. ['secrets']"
+}
+
+variable "aws_ssm_agent_enabled" {
+  type        = bool
+  description = "Set true to attach the required IAM policy for AWS SSM agent to each EC2 instance's IAM Role"
+  default     = false
+}
+
+variable "kubeconfig_file" {
+  type        = string
+  default     = ""
+  description = "Name of `kubeconfig` file to use to configure Kubernetes provider"
+}
+
+variable "kubeconfig_file_enabled" {
+  type        = bool
+  default     = false
+  description = <<-EOF
+    Set true to configure Kubernetes provider with a `kubeconfig` file specified by `kubeconfig_file`.
+    Mainly for when the standard configuration produces a Terraform error.
+    EOF
+}
+
+variable "aws_auth_yaml_strip_quotes" {
+  type        = bool
+  default     = true
+  description = "If true, remove double quotes from the generated aws-auth ConfigMap YAML to reduce spurious diffs in plans"
+}
+
+variable "cluster_private_subnets_only" {
+  type        = bool
+  default     = false
+  description = "Whether or not to enable private subnets or both public and private subnets"
+}
+
+variable "allow_ingress_from_vpc_stages" {
+  type        = list(string)
+  default     = ["auto", "corp"]
```

Comment:
IMO defaulting to no access and having this be part of the catalog would be a better choice